app/controllers/charges_controller.rb

```diff
@@ -0,0 +1,31 @@
+class ChargesController < ApplicationController
+
+  def new
+  end
+
+  def create
+
+    if session[:cart_id]
+      @cart = Cart.find(session[:cart_id])
+    end
+
+    # Amount in cents
+    @amount = (@cart.total*100).to_i
+
+    customer = Stripe::Customer.create(
+      :email => params[:stripeEmail],
+      :source  => params[:stripeToken]
```

Comment:
?? seems like working anyway, and that is the same as in the strip documentation setup